README.md

```diff
@@ -1,8 +1,9 @@
 # java-etherscan-api
 
-![travis](https://travis-ci.org/GoodforGod/java-etherscan-api.svg?branch=master)
+[![travis](https://travis-ci.org/GoodforGod/java-etherscan-api.svg?branch=master)](https://travis-ci.com/iSnow/java-etherscan-api)
```

Comment:
I think that this should be linked to https://jitpack.io/#goodforgod/java-etherscan-api as other links too